ci-operator/step-registry/openshift/microshift/infra/wait-for-cluster-up-aws/openshift-microshift-infra-wait-for-cluster-up-aws-commands.sh

```diff
@@ -1,11 +1,14 @@
 #!/bin/bash
 set -xeuo pipefail
 
+
 curl https://raw.githubusercontent.com/openshift/release/master/ci-operator/step-registry/openshift/microshift/includes/openshift-microshift-includes-commands.sh -o /tmp/ci-functions.sh
 # shellcheck disable=SC1091
 source /tmp/ci-functions.sh
 ci_script_prologue
 trap_subprocesses_on_term
+trap_install_status_exit_code $EXIT_CODE_WAIT_CLUSTER_FAILURE
```

Comment:
This overrides the subprocess TERM trap, so we need something that runs both, probably?